versioned_docs/version-2.4.0/Basic Usage/walkthough.md

```diff
@@ -10,7 +10,7 @@ To toggle file explorer you need to press `Space` + `e`
 
 ### Opening terminal
 
-To toggle terminal you need to press `Ctrl` + `\`
+To toggle terminal you need to press `Ctrl` + `'`
```

Comment:
Could you please change this to `F7` for maximum compatibility in the walkthrough?

Reply:
Done!